src/modules/sign/components/Signin.css

```diff
@@ -13,15 +13,15 @@
 	margin: 0 auto;
 }
 .sign_in_text {
-	font-family: Inter, serif;
+	font-family: sans-serif;
 	font-weight: 600;
 	font-size: 1.5rem;
 	line-height: 2.063rem;
 	margin-top: 4rem;
 	color: #37464f;
 }
 .more_text {
-	font-family: Inter, serif;
+	font-family: sans-serif;
```

Comment:
This can be removed ow since the document now has the same font-family

Reply:
Okay.